composer.json

```diff
@@ -10,8 +10,8 @@
         }
     ],
     "require": {
-        "php": "^7.4.3",
-        "symfony/http-client": "5.2.*",
+        "php": ">=7.4.3",
+        "symfony/http-client": "^5.2.8",
```

Comment:
Hi,

Unfortunately you cannot mix `^` and `*`.
It raises the error `Could not parse version constraint ^5.2.*: Invalid version string "^5.2.*" `
Because `5.2.*` means `>=5.2.0` and `<5.3.0`, the previous rule prevents installation on `5.3+`